File: fundamentals/fundamentals-lab/L04_Method/src/MathOperations.java
import java.util.Scanner;

public class MathOperations {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int firstNumber = Integer.parseInt(scan.nextLine());
        String operator = scan.nextLine();
        int secondNumber = Integer.parseInt(scan.nextLine());

        int result = calculateNumbers(firstNumber,operator,secondNumber);

        System.out.println(result);
    }

    private static int calculateNumbers(int firstNumber, String operator, int secondNumber) {
int calculatino = 0;
        switch (operator) {
            case "+":
                calculatino = firstNumber + secondNumber;
                break;
            case "-":
                calculatino = firstNumber - secondNumber;
                break;
            case "*":
                calculatino = firstNumber * secondNumber;
                break;
            case "/":
                calculatino = firstNumber / secondNumber;
                break;
        }
        return calculatino;
    }

}

File: js/advanced/js-advanced-lab/L01_SyntaxFunctionsStatemens/math-operations.js
function calculator(firstNumber, secondNumber, symbol) {
    let result;
    if (symbol == '+') {
        result = firstNumber + secondNumber;
    } else if (symbol == '-') {
        result = firstNumber - secondNumber;
    } else if (symbol == '*') {
        result = firstNumber * secondNumber;
    } else if (symbol == '/') {
        result = firstNumber / secondNumber;
    } else if (symbol == '%') {
        result = firstNumber % secondNumber;
    } else if (symbol == '**') {
        result = firstNumber ** secondNumber;
    }
    console.log(result);
}

calculator(2, 2, '**');